index.js

```diff
@@ -1,4 +1,14 @@
-const cardContainer = document.querySelector('.card-container');
+
+
+
+const cardContainer = document.getElementById('cardContainer');
+for (let i = 1; i <= 9; i++) {
+const listItem = document.createElement('li');
```

Comment:
Please add proper spacing/Indentation. 
Add preetier extension to vs code and try.